source/_integrations/elro_connects.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: Elro Connects
+description: Instructions on integrating Elro Connects with Home Assistant.
+ha_category:
+  - Siren
+ha_release: 2022.6.0
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@jbouwh'
+ha_domain: elro_connects
+ha_platforms:
+  - siren
+ha_integration_type: integration
+---
+
+The `elro_connects` integration will allow users to integrate their Elro Connects fire, heat, CO, water or smoke alarms as a siren entity into Home Assistant. The alarms can be tested (and silenced) through the siren turn on (and turn off) services. The integration discovers entities (automatically) through an Elro Connects K1 connector plug. The entity names assigned in the Elro Connects app will be synced as entity name in Home Assistant.
+
+{% include integrations/config_flow.md %}
+
+As a prerequisite the Elro K1 Connector needs to be already setup and the alarm devices need to be pre-registered on the Elro Connects K1 connector.
```

Comment:
This should be above the config flow? Would be nice to read about prerequisites before starting setting it up in Home Assistant.